python/lsst/ts/dsm/dsm_csc.py

```diff
@@ -124,6 +102,36 @@ def get_config_pkg():
         """
         return "ts_config_eas"
 
+    async def handle_summary_state(self):
+        """Handle things that depend on state.
+        """
+        self.log.debug(f"Current state: {self.summary_state}")
+        if self.disabled_or_enabled:
+            self.log.debug(f"Telemetry dir: {self.telemetry_directory}")
+            try:
+                self.telemetry_watcher.watch(path=self.telemetry_directory, flags=aionotify.Flags.CLOSE_WRITE)
+                loop = asyncio.get_running_loop()
+                await self.telemetry_watcher.setup(loop)
+            except ValueError:
+                # Watch already running
+                pass
```

Comment:
If you know which of the three lines in the try block will raise ValueError watch is already running consider just having that in the try block.

Reply:
The first and third calls both throw exceptions (ValueError and AssertionError respectively) on duplicates. I've added the other exception.